### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by mohamedSabry0**
A JavaScript project setup for a leaderboard list app, using Webpack and ES6 features, notably modules. I developed the first version of the app following a wireframe.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+  <h1 class="header">Leaderboard</h1>
+  <main class="container">
+    <div class="score-container">
+      <h3>Recent scores</h3>
+      <button type="submit" id="refresh">Refresh</button>
```
Comment (Ugboaja-Uchechi) [inline src/index.html]: - [ ] Great job 👍 . Kindly make sure the `h3` element and `button` are aligned on the same row.

WireFrame Design | Your Design
------------- | -------------
image |  image

File: src/index.html [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+      <ul class="scores-list"></ul>
+    </div>
+    <div class="input-container">
+      <h3>Add your score</h3>
+      <input type="text" id="name" value="name" placeholder="Your name">
+      <input type="text" id="score" value="score" placeholder="Your score">
+      <button type="submit" id="submit">Submit</button>
```
Comment (Ugboaja-Uchechi) [inline src/index.html:22]: - [ ] Kindly make sure the ` submit button` is smaller and looks as close as possible to the provided design

WireFrame Design | Your Design
------------- | -------------
image|  image

File: src/style.css [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+  display: flex;
+  justify-content: space-between;
 }
 
-body {
-  background-color: bisque;
+.scores-list {
+  padding: 0;
+  list-style: none;
+}
```
Comment (Ugboaja-Uchechi) [inline src/style.css]: - [ ] Kindly make sure the list has a `border-radius` around it 🙏 
- [ ] Kindly make sure each row in the list have alternating colors as seen in the Figma design 🙏 . You can use the CSS `:nth` selector for this. You can read more about it in this [article](https://css-tricks.com/almanac/selectors/n/nth-of-type/)

WireFrame Design | Your Design
------------- | -------------
image|  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @mohamedSabry0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @mohamedSabry0 , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**